databases/relational-databases.md

```diff
@@ -117,12 +166,25 @@ Schema design:
 Further learning: Object-Relational Mapping (& ORM libaries)
 - [freeCodeCamp: Which JavaScript ORM should you be using in 2018?](https://medium.freecodecamp.org/a-comparison-of-the-top-orms-for-2018-19c4feeaa5f)
 
-### Check for Understanding
-
+Solve the quizes mentioned below
 - [W3Schools quiz](http://www.w3schools.com/sql/sql_quiz.asp)
 - [JavaTPoint sql quiz](https://www.javatpoint.com/sql-quiz)
 - [TutorialsPoint quiz](https://www.tutorialspoint.com/sql/sql_online_quiz.htm)
 
+
+### Check for Understanding
+
```

Comment:
Thanks for those questions! I just have some grammar comments.

Reply:
Fixes look good.  Does the above outline provide answers to all these questions? Please look at each one.  If not, the info needs to be added.